src/index.ts

```diff
@@ -1,42 +1,56 @@
 import Client from './client';
+import LRU from 'lru-cache';
 import { Options, Target } from './types';
 import { Logger } from './log';
-import { KeyValueStore } from './types';
+import { AsyncKeyValueStore, KeyValueStore } from './types';
+import { FileStore } from './store';
 
-export { Client, Options, Target, KeyValueStore, Logger };
+export {
+  Client,
+  Options,
+  Target,
+  AsyncKeyValueStore,
+  KeyValueStore,
+  Logger,
+  LRU,
+  FileStore,
+};
 export default {
   instance: undefined,
   init: function (sdkKey: string, options: Options): void {
     if (!this.instance) {
       this.instance = new Client(sdkKey, options);
     }
   },
-  boolVariation: function (
+  boolVariation: async function (
     identifier: string,
     target: Target,
     defaultValue = false,
-  ): boolean {
+  ): Promise<boolean> {
     return this.instance.boolVariation(identifier, target, defaultValue);
   },
-  stringVariation: function (
+  stringVariation: async function (
     identifier: string,
     target: Target,
     defaultValue = '',
-  ): string {
+  ): Promise<string> {
     return this.stringVariation(identifier, target, defaultValue);
```

Comment:
This is a recursive call. Was it supposed to be...
```suggestion
    return this.instance.stringVariation(identifier, target, defaultValue);
```